tidb/assets/service_checks.json

```diff
@@ -1,18 +1 @@
-[
```

Comment:
Could you add it back? Even though it's not used it's still sent, so it should be documented